From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 30 Jul 2019 17:56:17 -0500
Subject: [PATCH] fix install os-core-plus

---
 isoutils/isoutils.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/isoutils/isoutils.go b/isoutils/isoutils.go
index ee49e5c..fce1c81 100644
--- a/isoutils/isoutils.go
+++ b/isoutils/isoutils.go
@@ -123,8 +123,8 @@ func mkInitrd(version string, model *model.SystemInstall, options args.Args) err
 	options.SwupdFormat = "staging"
 	sw := swupd.New(tmpPaths[clrInitrd], options)
 
-	/* Install os-core only as initrd */
-	if err := sw.VerifyWithBundles(version, model.SwupdMirror, []string{}); err != nil {
+	/* Install os-core and os-core-plus as initrd */
+	if err := sw.VerifyWithBundles(version, model.SwupdMirror, []string{"os-core-plus"}); err != nil {
 		prg = progress.NewLoop(msg)
 		prg.Failure()
 		return err
-- 
https://clearlinux.org

