From 9628afa9f0519134a1bf6bf5b83079f00f8c5776 Mon Sep 17 00:00:00 2001
From: Mark D Horn <mark.d.horn@intel.com>
Date: Wed, 7 Jul 2021 14:59:20 -0700
Subject: [PATCH 3/3] Force the version due to patching

Signed-off-by: Mark D Horn <mark.d.horn@intel.com>
---
 Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile b/Makefile
index 555e917b2e0d..4e8fd13056f4 100644
--- a/Makefile
+++ b/Makefile
@@ -21,6 +21,7 @@ else
 export CGO_CFLAGS = $(shell go env CGO_CFLAGS) -Wno-deprecated-declarations
 endif
 
+export VERSION = 2.7.2-2-g9eac32df5b82
 export GOPATH=$(pkg_dir)
 export GO_PACKAGE_PREFIX := github.com/clearlinux/clr-installer
 export TESTS_DIR := $(top_srcdir)/tests/
@@ -76,6 +77,8 @@ VERSION=$(shell basename $(mkfile_dir) | awk -F- '{print $$NF}')
 endif
 endif
 
+# Force to disable version check due to patching
+IS_GIT_REPO=1
 # Validate the Version
 validate_version:
 ifeq ($(IS_GIT_REPO),0)
-- 
2.32.0

